math-doc/math/scribblings/math-matrix.scrbl

```diff
@@ -268,9 +274,19 @@ Returns an array with two-dimensional arrays @racket[Xs] along the diagonal and
                                                                            (error "this procedure should never be called")))
                                               (col-matrix [1 2 3])
                                               (row-matrix [4 5 6])))
-                 (block-diagonal-matrix (list (make-matrix 2 2 2.0+3.0i)
-                                              (make-matrix 2 2 5.0+7.0i))
-                                        0.0+0.0i)]
+                 (block-diagonal-matrix (list (build-simple-array #(2 0) (λ: ([js : Indexes])
```

Comment:
Definitely better. Instead of passing a function raising an error, I think in this example we can just do `(const 1)` or `(const +nan.0)`, right? It'll still be an array of numbers in either case because the off-diagonal elements will be zeros. Should make the example much shorter.